docs/UserGuide.adoc

```diff
@@ -813,8 +861,8 @@ The following sections summarize the commands you can use in contactHeRo.
 [width="99%",cols="33%,33%,33%",options="header",]
 |===
 |Function |Command |Example
-|Adding a person. |`add n/NAME p/PHONE e/EMAIL a/ADDRESS cp/CURRENT_POSITION cc/COMPANY [pp/PROFILE PICTURE NAME] [s/SKILL]...` |`add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 cp/Software Engineer cc/Google pp//home/trafalgarandre/downloads/john.jpeg s/Java s/C++`
-|Editing an existing person. |`edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [cp/CURRENT_POSITION] [cc/COMPANY] [pp/PROFILE PICTURE NAME] [s/SKILL]...` |`edit 1 p/91234567 e/johndoe@example.com`
+|Adding a person. |`add n/NAME p/PHONE e/EMAIL a/ADDRESS cp/CURRENT_POSITION cc/COMPANY [pp/PROFILE_PICTURE_PATH] [s/SKILL]...` |`add n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 cp/Software Engineer cc/Google pp//home/trafalgarandre/downloads/john.jpeg s/Java s/C++`
```

Comment:
isn't it PROFILE_PICTURE_PATH?

Reply:
? haven't updated from Profile picture name to profile_picture_path

Reply:
Only for UG and result display, change it to path?